content/docs/fundamentals/understanding_architecture.md

```diff
@@ -0,0 +1,112 @@
+---
+weight: 310
```

Comment:
Let's include weight to file name to simplify review and finding the right file.

Reply:
File names are translated to URL path. Looks ugly
![image](https://github.com/user-attachments/assets/df69a7af-70f7-4cf3-b958-5bb6dc29d4c6)